Templates/DefaultProject/Template/Registry/user.setreg

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I usually don't check in anything with "user" in the name. Maybe something like "loadlevel.setreg" or "autoloadlevel.setreg"